src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,45 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, currentTab, onTabSelected }) => {
+  let selectedTab = tabs.find(tab => tab.id === currentTab.id) || tabs[0];
+
+  if (selectedTab === undefined) {
+    // eslint-disable-next-line prefer-destructuring
+    selectedTab = tabs[0];
+  }
```

Comment:
You have already set in previous line of the code that if left operand returns `false` it set `tabs[0]` as a default selected tab
So you should remove it